### PR TITLE
worker benchmarks

### PR DESCRIPTION
**Pull request by afinch7**

### CODE REVIEW

File: tests/workers_round_robin_bench.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+const data = "HTTP/1.1 200 OK\r\nContent-Length: 12\r\n\r\nHello World\n";
```
Comment (ry) [inline tests/workers_round_robin_bench.ts]: Can you give some summary comment at the top of this file explaining what this measures?

File: tests/workers_round_robin_bench.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+const workerCount = 4;
+const cmdsPerWorker = 400;
+
+export interface ResolvableMethods<T> {
+  resolve: (value?: T | PromiseLike<T>) => void;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  reject: (reason?: any) => void;
+}
+
+export type Resolvable<T> = Promise<T> & ResolvableMethods<T>;
```
Comment (ry) [inline tests/workers_round_robin_bench.ts]: can this not be imported from elsewhere?

File: tools/benchmark.py [hunk 3]
```diff
@@ -25,6 +25,8 @@
     ("error_001", ["tests/error_001.ts"]),
     ("cold_hello", ["tests/002_hello.ts", "--reload"]),
     ("cold_relative_import", ["tests/003_relative_import.ts", "--reload"]),
+    ("workers", ["tests/workers_startup_bench.ts"]),
+    ("workers_round_robin", ["tests/workers_round_robin_bench.ts"]),
```
Comment (ry) [inline tools/benchmark.py]: 👍

File: tests/workers_round_robin_bench.ts (hunk 2 above)
Comment (afinch7) [reply tests/workers_round_robin_bench.ts]: It only exists within `//js/util.ts`. Maybe std needs some tools for implementing async messaging channels like this.

**Pull request merged**